webpack/src/index.js

```diff
@@ -0,0 +1,29 @@
+import './style.css';
+
+const itemList = [];
+const storeList = document.getElementById('listItem');
+
+// let removeIndex = storeList.children;
+
+const showItem = () => {
```

Comment:
- [ ] Your _js_ module does not include `array object` of `tasks`. Kindly ensure that it has an array of some simple to do tasks (array of objects) with each object containing three keys - `description [string]`, `completed [bool]`, `index: [number]`. 
- [ ] Also, please ensure that on _page load_, the task array is rendered **dynamically** to the DOM. The tasks should appear in order of the `index` values for each task

Reply:
Thanks for the review. It took a whole day of study to implement though 😩😩